### DISCUSSION TITLE
v2.8.75

### DISCUSSION THREAD

**Discussion by fiftin** [semaphoreui/semaphore#1094] [Announcements]
[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/L4L4FKF76)

## Fixed LDAP bugs

Issues by LDAP available in the project [Fix LDAP](https://github.com/ansible-semaphore/semaphore/projects/22).

Thanks to [@cptsanifair](https://twitter.com/cptsanifair) we solved LDAP issues that we managed to reproduce.

Added [docs](https://docs.ansible-semaphore.com/administration-guide/ldap) for LDAP configuration. Fill free to create PRs for the docs.

## Implemented integration tests for all supported databases

New all 3 supported databases (BoltDB, Postgres, MySQL) are tested on integration tests.

Thanks to **se-di** for support! 

## Semaphore no more blocks BoltDB file

To make integration tests possible for BoltDB we had to implement session connection to BoltDB file. Semaphore blocks the file only when read or write to it.

I think it is very useful feature for other applications too.

<hr /><em>This discussion was created from the release <a href='https://github.com/ansible-semaphore/semaphore/releases/tag/v2.8.75'>v2.8.75</a>.</em>

**Comment by leandroscardua** [comment]
It`s working well with FreeBSD :D

Thank you for your support to releasing FreeBSD version
